src/library/export/trackexportdlg.cpp

```diff
@@ -66,30 +67,34 @@ void TrackExportDlg::slotAskOverwriteMode(
         std::promise<TrackExportWorker::OverwriteAnswer>* promise) {
     QMessageBox question_box(
             QMessageBox::Warning,
-            tr("Overwrite Existing File?"),
-            tr("\"%1\" already exists, overwrite?").arg(filename),
+            tr("Replace Existing File?"),
+            tr("\"%1\" already exists, replace?").arg(filename),
             QMessageBox::Cancel);
 
     QPushButton* pSkip = question_box.addButton(
             tr("&Skip"), QMessageBox::NoRole);
-    QPushButton* pSkipAll = question_box.addButton(
-            tr("Skip &All"), QMessageBox::NoRole);
     QPushButton* pOverwrite = question_box.addButton(
-            tr("&Overwrite"), QMessageBox::YesRole);
-    QPushButton* pOverwriteAll = question_box.addButton(
-            tr("Over&write All"), QMessageBox::YesRole);
+            tr("&Replace"), QMessageBox::YesRole);
     question_box.setDefaultButton(pSkip);
 
+    QCheckBox* pApplyToAll = new QCheckBox(tr("Apply to all files"));
```

Comment:
You might need to add a parent to `question_box` otherwise it would leak IIRC. Or you could also delete explicitly